<launch>
    <arg name="lim_x" default="11" />
    <arg name="lim_y" default="11" />
    <arg name="lim_theta" default="3.141592" />


    <node name="send_goal" 
        pkg="turtle_infnet" 
        type="send_goal.py"
        args="$(arg lim_x) $(arg lim_y) $(arg lim_theta)"
        output="screen">

	<remap from="goal" to="/goal"/>
	<remap from="control_type" to="/turtle1/control_type"/>
	</node>
	
</launch>
